apps/firelens-stability/collections/temp-golden-path-high-throughput-only/suite-config.json

```diff
@@ -0,0 +1,10 @@
+{
+    "definitions": {
+        "testId": "{{managed.suiteName}}-{{managed.caseName}}",
+        "segfault": false
+    },
+    "config": {
+        "cluster": "{{managed.executionName}}-{{managed.executionId}}-{{managed.collectionName}}",
```

Comment:
if we create a new cluster each time, should it include the data??

Reply:
answer: verbal explanation given that data is in the execution name

Reply:
I mean `DATE` not data

Reply:
why not have the ID just be a timestamp?

Reply:
wait if execution name is user inputted, then how do we know it has a date? that's just a convention?

Reply:
I wrote it down wrong ID is output and includes a date